internal/services/firewall/firewall_resource_test.go

```diff
@@ -194,6 +195,31 @@ func TestAccFirewall_withZones(t *testing.T) {
 	})
 }
 
+func TestAccFirewall_skuTier(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_firewall", "test")
+	r := FirewallResource{}
+	skuTier := string(network.AzureFirewallSkuTierStandard)
+	skuTierUpdate := string(network.AzureFirewallSkuTierPremium)
```

Comment:
can we just use constants here instead of pulling the value from the sdk?

Reply:
@katbyte I pushed all the changes that you mentioned. Thanks!